host-src/tool/dc-tool.c

```diff
@@ -281,7 +284,11 @@ char serial_getc()
     retval = serial_read(&tmp, 1);
     if (retval == -1) {
         printf("serial_getc: read error!\n");
-        tmp = 0x00;
+#ifndef __MINGW32__		
+		tmp = (char)NULL;
```

Comment:
I don’t remember why this was necessary to do so, just init to 0x0 should be enough

Reply:
Oops. Something I forgot to bring over from a previous pr. I'll fix it when I get home.

Reply:
This change was made in one of the commits in this pull request. Your comments is on a older commit in this pull request. You want to see "Changes from all commits"